Makefile

```diff
@@ -35,6 +35,25 @@ clean:
 htmlproofer:
 	@echo "Do not make docs changes against this branch, please use master."
 
+###############################################################################
+# CI / test targets 
+###############################################################################
+
+ci: htmlproofer kubeval
+
+kubeval:
+	# Run kubeval to check master manifests are valid Kubernetes resources.
+		docker run -v $$PWD:/calico --entrypoint /bin/sh -ti garethr/kubeval:0.1.1 -c 'ok=true; for f in `find /calico/_site/master -name "*.yaml" |grep -v "\(config\|allow-istio-pilot\|30-policy\).yaml"`; do echo Running kubeval on $$f; /kubeval $$f || ok=false; done; $$ok'
```

Comment:
This is checking in `_site/master` when I think it should be `_site/v2.6` for this branch?

Reply:
fixed.